### ISSUE TITLE
Cannot disable paredit without disabling calva

### ISSUE THREAD

**Issue by epiphone** [BetterThanTomorrow/calva-paredit#10]
Thanks for a great plugin! I'd like to use Calva without Paredit, any chance for that to happen? Now when trying to disable Calva Paredit, VSCode throws the following error:

```
Cannot disable extension 'Calva Paredit'. Extension 'Calva: Clojure & Clojurescript interactive programming made easy' depends on this.
```

I'm on vscode 1.28.0, Calva 1.3.44 and Calva Paredit 0.1.15.

**Comment by PEZ** [comment]
Hello. It would go a bit against the goals of Calva, to not bundle Paredit, but I might consider it anyway. (Might being tje key word there :smile:) Can you share with me why you do not want it installed with Calva (or rather, why you would want to be able to uninstall it)?

**Comment by epiphone** [comment]
It's because Paredit overwrites my navigation shortcuts, which is a nuisance especially when editing other than Clojure files in the same project. A stopgap solution for now is to manually delete all Paredit key bindings, but I imagine there might be others too who want to use the fantastic Calva plugin without necessarily going for Paredit.

**Comment by PEZ** [comment]
That was not good! I have now released a version, `0.1.16`, that tries to address the problem in two ways:

1. Only activating the default keybindings with clojure and lisp files.
1. Adding a configuration option which disables all default keybindings in one go.

Let me know if this is helpful for your case.

**Comment by epiphone** [comment]
That's brilliant, thank you very much!
